connectivity/drivers/ble/FEATURE_BLE/COMPONENT_BlueNRG_2/README.md

```diff
@@ -11,25 +11,62 @@ A simple table like this could help:
 |BlueNRG-M2SP |BlueNRG-2 (application processor) |v5.0 |Active (included in ST's Longevity Program) |X-NUCLEO-BNRG2A1 | https://www.st.com/en/wireless-transceivers-mcus-and-modules/bluenrg-m2.html |
 
 
-The library uses ARM Cordio stack. It does not work with the stock firmware that is loaded in the BLE module of the X-NUCLEO-BNRG2A1 expansion board.
+The library uses ARM Cordio stack. It does not work with the stock firmware that is loaded in the BLUENRG-M2SP BLE module of the X-NUCLEO-BNRG2A1 expansion board.
 
-In order to use this library with X-NUCLEO-BNRG2A1, you need to replace the firmware of the BLUENRG-M2SP BLE module mounted on that expansion by using one of these binaries (provided for both STM32 NUCLEO-L476RG and STM32 NUCLEO-F401RE):
+In order to use this library with X-NUCLEO-BNRG2A1, you need to update the firmware of the BLUENRG-M2SP BLE module mounted on that expansion.
 
-[BlueNrg2FwUpdater_L4.bin](fwupdater/BlueNrg2FwUpdater_L4.bin)
+To this aim, attach the X-NUCLEO-BNRG2A1 to the STM32 NUCLEO board through the Arduino Connector. You can use the binary provided for both STM32 NUCLEO-F401RE and STM32 NUCLEO-L476RG.
 
-[BlueNrg2FwUpdater_F4.bin](fwupdater/BlueNrg2FwUpdater_F4.bin)
+Download the binaries from the links below:
+
+[BLE2_StackUpdater_F401RE.bin](https://github.com/STMicroelectronics/mbed-ble/BlueNRG2_StackUpdater/BLE2_StackUpdater_F401RE.bin)
+
+[BLE2_StackUpdater_L476RG.bin](https://github.com/STMicroelectronics/mbed-ble/BlueNRG2_StackUpdater/BLE2_StackUpdater_L476RG.bin)
 
 Before flashing your target board with the respective binary, be careful to properly set the SPI reset on pin **D7** as reported in the picture below:
 
 ![D7_SPI_rst](img/D7_SPI_rst.jpg)
 
 At the end of the updating procedure, the LED labeled as **LD2** on the STM32 NUCLEO board will be blinking.
+You can also check the status of the updating procedure on the serial terminal (baud rate set to 115200).
+The picture below shows the log upon success:
+
+![updLog](img/updLog.jpg)
+
+If, for some reason, the updating procedure does not succeed you can use a standard ST-Link V2 with 5 jumper wires female-female together with the
+[BlueNRG2_FlasherUtility](https://github.com/STMicroelectronics/mbed-ble/BlueNRG2_FlasherUtility/en.STSW_BNRGFLASHER.zip) software tool (currently available only for Windows PC).
+
+You need to connect the J12 pins of the X-NUCLEO-BNRG2A1 to the pins of the ST-Link V2 as shown in the picture below:
+
+![ST-LINK](img/ST-LINK.jpg)
+
+In particular we have the following connections:
+
+|    |J12|ST/Link V2|
+|--------|----------|
+|Pin |1  |1         |
+|Pin |2  |9         |
+|Pin |3  |12        |
+|Pin |4  |7         |
+|Pin |5  |15        |
+
```

Comment:
what about the SPI reset on pin D7?

Reply:
Did you follow the steps indicated (also by pictures) in the updated README?

Reply:
Hello, yes, the readme has the same instruction as the document formerly linked to. It mentions the d7 when flashing using the boards but not when using st-link

Reply:
Please tell me if the D7 pin should be connected by the jump-wire when flashing using st-link like when flashing through the board.

Reply:
all sorted now using the the correct version of st brng flasher